### ISSUE TITLE
Functionality to support specifying language for foreign words

### ISSUE THREAD

**Issue by yblock** [openstax/cnx-recipes#4837]
# Brief Description of Scenario
The goal here is for screen readers to recognize and pronounce foreign words correctly, we need to provide the appropriate language tag for a foreign word. 
Going off of documentation [here](https://www.w3.org/International/articles/language-tags/index.en) and [here](https://www.w3.org/WAI/WCAG22/Understanding/language-of-parts.html) for Language of Parts and Language tagging

# Acceptance Criteria
2 options to think about:
1. If possible, we could simply add the country/language short code like "sp" for Spanish to the foreign tag class. That would end up looking like: `<foreign class="no-emphasis sp">bueno</foreign>` If it's possible to just target the language portion of the class to extract and transform to the lang tag. 
2. If that's not possible, maybe we can request that 'lang=""' be added as an acceptable child to '<foreign>' in POET validation. In which case the cnxml would look like `<foreign class="no-emphasis" lang="sp">bueno</foreign>` 

For both cases, after an HTML transform, we would want it to look like `<span class="no-emphasis" lang="sp">bueno</span>`

# Sample Mark-Up
```
<foreign class="no-emphasis sp">bueno</foreign>
or
<foreign class="no-emphasis" lang="sp">bueno</foreign>
transformed into
<span class="no-emphasis" lang="sp">bueno</span>
```

# Location(s)
- **osbooks repo:**  https://github.com/openstax/osbooks-business-law/commits/foreign-tag-test
- **collection slug:** business-law-i-essentials
- **ref/branch:** foreign-tag-test
- **chapter/section:**  m78920
- **surrounding text:**  <para id="fs-idm334729840">
"Laissez faire"

**Comment by lwtchu** [comment]
Phil's PHILosophy is to use classes for styling and other attributes for other data.

**Comment by lwtchu** [comment]
@wilkus27 to check if polish books are using name spaces instead of lang attribute

**Comment by wilkus27** [comment]
@lwtchu We are using attributes `xml:lang="en"` and `lang="en"`, but as we checked it doesn't work in ChromeVox :disappointed:

**Comment by lwtchu** [comment]
@wilkus27 hmmm then we need to figure out what HTML it should transform into and reverse engineer it. 🤔 need to think about approach

**Comment by lwtchu** [comment]
@wilkus27 are the attributes added in the CNXML or via recipe?

**Comment by wilkus27** [comment]
@lwtchu As I see we are adding first attribute in cnxml:

`<foreign xml:lang="en"><term cxlxt:index="foreign">dissertation</term></foreign>`

and a second one is created during transformation from cnxml to xhtml:

![image](https://user-images.githubusercontent.com/40228854/187611711-244943b3-c884-4057-8610-c4b92881d9e8.png)


Also, per Krzysztof, this is a correct way of supporting foreign words in xml, and if it doesn't work it's probably on the screen reader side, so there's nothing what we can really do about it :disappointed:

**Comment by lwtchu** [comment]
@wilkus27 i  talked to alan and for some screen readers it has to be enabled.

@yblock do you mind adding it to a sample in business law when you get the chance and generate in CORGI (web preview) and see if it shows up in the tagging?

**Comment by yblock** [comment]
Here's the HTML output when formed like `<foreign class="no-emphasis" xml:lang="fr">Laissez faire</foreign>`

`<span data-type="foreign" class="no-emphasis" xml:lang="fr" lang="fr">Laissez faire</span>`

**Comment by lwtchu** [comment]
@yblock ok i think if the `lang` attribute shows up, the jaws will know to read it in french. based on [this thread](https://openstax.slack.com/archives/C4PJYP0LF/p1661874348868879) in the a11y channel 

I'm going to close out this  card...but if you add the `XML:lang` to words it should work.
